themes/default/content/docs/concepts/vs/terraform/_index.md

```diff
@@ -25,7 +25,7 @@ aliases:
     }
 </style>
 
-Choosing the right infrastructure as code platform is critical. We want you to have the best information possible to pick the solution that fits your needs.
+Pulumi and Terraform are both infrastructure as code technologies. Pulumi is open source and offers the flexibility to use any programming language for managing infrastructure. This makes it widely accessible to developers and devops engineers from any background. Terraform requires the use of a proprietary domain-specific language (DSL) and is not open source. In this comprehensive guide, we'll explore the key differences, similarities, and real-world scenarios, mapping Terraform concepts to Pulumi to help you choose the right infrastructure as code platform.
```

Comment:
Obviously this info's super relevant, but it feels a bit "wedged into the middle" of this introductory paragraph to me. Seems like it belongs in the key differences paragraph instead.

```suggestion
Pulumi and Terraform are both infrastructure as code technologies. In this comprehensive guide, we'll explore their key differences, similarities, and real-world scenarios, mapping Terraform concepts to Pulumi to help you choose the right infrastructure as code platform.
```

If we did want to keep it here, we might just call it out briefly as a "for example", like "Pulumi and Terraform are alike in many ways, but different in many others. Pulumi, for example, is open-source, whereas Terraform is not. In this comprehensive guide, we'll cover these and other key differences in depth..." ... or something. 😄